_studio/shared/include/mfx_mfe_adapter.h

```diff
@@ -27,10 +27,9 @@
 #include <map>
 #include <list>
 #include <vector>
-#include "vm_mutex.h"
-#include "vm_cond.h"
 #include "vm_time.h"
 #include <mfxstructures.h>
+#include <condition_variable>
```

Comment:
Please, move cv header closer to list and vector. And check whether you really need vm_time.h, I believe you don't.